ls++

```diff
@@ -362,6 +387,10 @@ sub perm_size_file {
 
 sub perm {
   my ($perm) = @_;
+  if (length($perm) < $permlen){
+    # adding an extra space to align with occasional +
+    $perm .= " ";
```

Comment:
Extra padding when necessary: 
![](http://i.imgur.com/yIqBVGI.png)

When not needed: 
![](http://i.imgur.com/TE7Y1Lc.png)